src/VoodooManager.ts

```diff
@@ -0,0 +1,108 @@
+// TODO: this is a hacky wrapper class for a Voodoo contact,
```

Comment:
We should probably make a `voodoo` folder to contain this and all the other new classes we are going to be creating.

Reply:
Missed this, I'll make it happen